jobset.nix

```diff
@@ -0,0 +1,16 @@
+{ nixpkgs }:
```

Comment:
I think we should move this into the nixos subdirectory.

Reply:
This is just a temporary thing to be deleted before it's merged. I'm using this file to have a CI for the four VM tests on my personal Hydra (see https://hydra.ist.nicht-so.sexy/jobset/nixpkgs/networkd-containers).